.golangci.yml

```diff
@@ -1,2 +1,20 @@
-run:
-  timeout: 5m
```

Comment:
can you explain why this was removed? if accidental, please add it back to ensure we still have timeout protection

Reply:
You're right, I updated `golangci-lint` to v2 and accidentaly removed it. It's added again now.